cmd/activity-monitor/main.go

```diff
@@ -39,28 +41,46 @@ const (
 	AmqpImmediate    = false
 )
 
-var openCalls int64
+var tMu sync.Mutex
```

Comment:
Please add this and deliveryTimings to a struct and pass it around to take the lock so that it's easier to tell what it's being user for.

Then add a Get, Add, and Delete methods to the struct that takes the locks for us so we don't have to keep track of them everywhere.